resources/utility-scripts/test_e2e.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+variantNameGetValuesValue="5238"
+variantNameGetValuesResult="$(printf "16:75238144:C>C\n6:52380882:G>G")"
+proteinChangeGetValuesValue="g32"
+proteinChangeGetValuesResult="$(printf "G325R\nG32E")"
+proteinChangeGetValuesValue2="7cfs*"
+proteinChangeGetValuesResult2="S137Cfs*28"
+hugoGeneSymbolGetValuesValue="tr5"
+hugoGeneSymbolGetValuesResult="HTR5A"
+
+variantNameGetVariantsValue="16:75238144:C>C"
+variantNameGetVariantsResult="-4530899676219565056"
+proteinChangeGetVariantsValue="G325R"
+proteinChangeGetVariantsResult="-2429151887266669568"
+hugoGeneSymbolGetVariantsValue="HTR5A"
+hugoGeneSymbolGetVariantsResult1="$(printf -- "-7039476204566471680\n-7039476580443220992\n-7039476780159200256")"
+hugoGeneSymbolGetVariantsResult2="$(printf -- "-7039476204566471680\n-7039476580443220992")"
+hugoGeneSymbolGetVariantsResult3="-7039476780159200256"
+
+query1="1 AND 2"
+resultQuery1="$(printf -- "count\n8\n8\n8")"
+query2="6 OR 16 AND 8"
+resultQuery2="$(printf -- "count\n1\n1\n1")"
+query3="5 AND 10 AND 15"
+resultQuery3="$(printf -- "count\n3\n3\n3")"
+query4="4 OR 11 OR 17"
+resultQuery4="$(printf -- "count\n5\n5\n5")"
+query5="3 OR 6 AND 9 AND 12 OR 15"
+resultQuery5="$(printf -- "count\n2\n2\n2")"
+
+test1 () {
+  docker-compose -f docker-compose.tools.yml run medco-cli-client --user test --password test $1 $2
```

Comment:
The credentials used should ideally be configurable. Is it possible to define and pass them through the .travis.yml file maybe?